src/modules/displayBooks.js

```diff
@@ -0,0 +1,46 @@
+const apiUrl = 'https://openlibrary.org/authors/OL23919A/works.json?limit=9';
+
+const displayBook = async () => {
+  try {
+    const response = await fetch(apiUrl);
+    const data = await response.json();
+
+    const mainSection = document.querySelector('.main-section');
+
+    data.entries.forEach((entry) => {
+      const { title, covers } = entry;
+
+      if (!covers || covers.length === 0) {
+        return;
+      }
```

Comment:
I kindly suggest that you put the fetch code into a separate function that can be called and exported whenever needed.